### ISSUE TITLE
Question about the reprojected distance loss function

### ISSUE THREAD

**Issue by zxiaomzxm** [TRI-ML/packnet-sfm#183]
Hi!
I have reimplement the reprojected distance loss in my own framework, 
but I found it hard to converge, because my initial pose is close to identity, thus in this scenario, the reprojected loss term is close to 0, right？Thus this scenario is a local minimal for this loss, it backpropagate to the pose directly, makes the photomatic loss diverged, and I can not get a good depth map in this setting.
Am I missing something?
